components/dashboard/src/projects/Project.tsx

```diff
@@ -259,6 +262,31 @@ export default function () {
                                     <span>Prebuild</span>
                                 </ItemField>
                             </Item>
+                            {isConsideredInactive && (
+                                <Alert
+                                    type={"warning"}
+                                    onClose={() => {}}
+                                    showIcon={true}
+                                    className="flex rounded mb-2 w-full"
+                                >
+                                    Prebuilds have been paused for this projects because no workspaces were opened for
+                                    the past 7+ days.{" "}
```

Comment:
**nitpick**: Minor typo in case this has been missed, and some ideas on rephrasing to include some of the benefits of this feature, and more:

> To reduce resource usage, **prebuilds** are automatically paused when not used for a workspace after 7 days.

Reply:
Changed.